config.js

```diff
@@ -35,6 +36,10 @@ const DEFAULT_DB_CONNECTION_POOL = { // sequelize defaults
   idle: 10000,
 };
 
+// Enable Rooms Routes
+const swagger = featureToggles.isFeatureEnabled('enableRooms') ?
+  './api/v1/rooms-swagger.yaml' : './api/v1/swagger.yaml';
```

Comment:
If broke up the swagger doc into a smaller ref models, then we could fix that problem.

Wont this feature flag be reversed once we get to the point where rooms features can be exposed?